<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIPCVideoFrame</class>
 <widget class="QFrame" name="CIPCVideoFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>远程管理</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>681</width>
     <height>491</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabVideo">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>所有视频</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidgetVideo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>271</height>
      </rect>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabGate">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="title">
     <string>开关闸</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>401</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>通道名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CAN地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开关闸</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>440</y>
       <width>320</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>开选定闸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>关选定闸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenAll">
        <property name="text">
         <string>开所有闸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCloseAll">
        <property name="text">
         <string>关所有闸</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
